Assets/Scripts/RobotControllerScript.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class RobotControllerScript : MonoBehaviour {
+
+    //variables that need to be set within Unity
+    [SerializeField]
+    private float maxSpeed = 10f; private float jumpForce = 5f; private float gravityForce = 7f;
+
+    [SerializeField]
+    private Rigidbody2D rb2d;
+
+    [SerializeField]
+    private Animator anim;
+
+    [SerializeField]
+    private Transform groundCheck;
+
+    [SerializeField]
+    private LayerMask whatIsGround;
+
+    [SerializeField]
+    private PhysicsMaterial2D playerMovingPhysicsMaterial, playerStoppingPhysicsMaterial;
+
+    [SerializeField]
+    private Collider2D playerGroundCollider;
```

Comment:
Consider getting the above Unity components with GetComponent<*SpecificComponent*>()

Reply:
There's the way to write it, thank you.

Reply:
changed as many as I could about 4 - 5 of them i think.